runtime-common/core/core-types/decl/mixed_decl.inl

```diff
@@ -222,6 +285,15 @@ public:
   const char *get_type_c_str() const;
   const char *get_type_or_class_name() const;
 
+
+  template<typename T>
+  inline bool empty_on(T key) const;
```

Comment:
Can we give a better name? Now it's a little confusing

Reply:
Could you give a tip?
This methods "checks that item that's stored by `key` index is empty". `empty_on/empty_at` are good for me, but you may advise something better